### DISCUSSION TITLE
Add Quiz or FAQs at the end of each chapter.

### DISCUSSION THREAD

**Discussion by BekahHW** [open-sauced/intro#209] [Ideas]
It would be nice to have a section at the end of each chapter that asks a couple of questions and summarizes the answers.

Something like [this](https://learnprompting.org/docs/basics/embeds):

<img width="786" alt="image" src="https://github.com/open-sauced/intro/assets/34313413/cf49f66d-a4b1-44dc-8633-7afc1497eed0">

**Comment by CBID2** [comment]
Good idea @BekahHW! :) I think quizzes would be best. It'll encourage users to think about what they just learned. Now, my question is how can we track the quizzes’ data to ensure that users understand the content(for example, 35% of people are not doing well on the quiz for the chapter on writing effective commit messages.)

**Comment by adiati98** [comment]
I think this would be helpful to solidify the learning. I've done a similar thing by adding a "mini challenge" in [one of my blog posts for OpenSauced](https://dev.to/opensauced/streamline-your-contributions-mastering-issue-forms-and-pr-templates-36j5).

But I think we might not need to have a quiz in each chapter. For example, in the [What is Open Source](https://intro.opensauced.pizza/#/intro-to-oss/what-is-open-source) and [Why Open Source](https://intro.opensauced.pizza/#/intro-to-oss/why-open-source) chapters. It would be more beneficial to have a quiz section in the chapters that hold more technical and practical topics.

One thing that crossed my mind at the moment is to create discussion(s) for folks to write down their answers and give them the opportunity to discuss their answers with each other and learn together. We can do this in a channel Discord, but I think discussions are more open and transparent and allow more folks to chime in. We can promote the discussion board(s) in Discord or social media or whatever if necessary.
